src/components/DuosHeader.js

```diff
@@ -27,6 +29,13 @@ class DuosHeader extends Component {
       return prev;
     });
   };
+   
+  SupportRequestModal = (e) => {
```

Comment:
nit: lower case the `S` for consistent method naming.
```suggestion
  supportRequestModal = (e) => {
```